### PR TITLE
Added supplementary page translations.

### PR DESCRIPTION
**Pull request by dbgoaco**
[DP-789](https://noticingsystem.atlassian.net/jira/software/projects/DP/boards/22?selectedIssue=DP-789)

### CODE REVIEW

File: Frontend/CO.CDP.OrganisationApp/Pages/ConfirmationMessage.cshtml [hunk 1]
```diff
@@ -1,7 +1,11 @@
 @page "/confirmation-message"
 
+@using CO.CDP.Localization
+@using Microsoft.AspNetCore.Mvc.Localization
+@inject IHtmlLocalizer<StaticTextResource> Localizer
+
 @{
-    ViewData["Title"] = "Thank you for contacting GOV.UK";
+    ViewData["Title"] = @StaticTextResource.Supplementary_Title;
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/ConfirmationMessage.cshtml:8]: The @ sign on @StaticTextResource is redundant as you're already inside a code block

**Review by andymantell — CHANGES_REQUESTED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/ConfirmationMessage.cshtml [hunk 2]
```diff
@@ -1,7 +1,11 @@
 @page "/confirmation-message"
 
+@using CO.CDP.Localization
+@using Microsoft.AspNetCore.Mvc.Localization
+@inject IHtmlLocalizer<StaticTextResource> Localizer
```
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ConfirmationMessage.cshtml:5]: Done.

File: Frontend/CO.CDP.OrganisationApp/Pages/ConfirmationMessage.cshtml (hunk 1 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ConfirmationMessage.cshtml:8]: Good spot, done.

File: Frontend/CO.CDP.OrganisationApp/Pages/ContactUs.cshtml [hunk 3]
```diff
@@ -99,13 +103,13 @@
...
-                                You have 10,000 characters available
+                                @StaticTextResource.Supplementary_ContactUs_CharsAvailable
                             </div>
                         </div>
                     </fieldset>
 
                     <govuk-button>
-                        Send
+                        @StaticTextResource.Supplementary_ContactUs_Send
                     </govuk-button>
```
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ContactUs.cshtml:112]: Done.

**Review by andymantell — APPROVED** [review]

**Review by dpatel017 — APPROVED** [review]

**Review by dharmverma — APPROVED** [review]

**Pull request merged**